localstack/services/sqs/sqs_listener.py

```diff
@@ -200,6 +226,19 @@ def forward_request(self, method, path, data, headers):
             elif action == 'DeleteQueue':
                 QUEUE_ATTRIBUTES.pop(_queue_url(path, req_data, headers), None)
 
+            elif action == 'ListDeadLetterSourceQueues':
+                queue_url = _queue_url(path, req_data, headers)
+                headers = {'content-type': 'application/xhtml+xml'}
+                content_str = _list_dead_letter_source_queues(QUEUE_ATTRIBUTES, queue_url)
+
+                new_response = Response()
```

Comment:
we can simplify lines 234-238 by adding an import at the top of the file and using `requests_response`:
```
from localstack.utils.aws.aws_responses import requests_response
...

return requests_response(content_str, headers=headers)
```